knimin/lib/data_access.py

```diff
@@ -804,7 +810,8 @@ def pulldown(self, barcodes, blanks):
                     blanks_copy['ANONYMIZED_NAME'] = blank
                     blanks_copy['HOST_SUBJECT_ID'] = blank
                     survey_md.append('\t'.join([blank] +
```

Comment:
if you hit enter right after the `append(` it will still be flake8 and give you much more room on the next line